packages/vite-plugin-cloudflare/playground/cloudflare-env/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@playground/cloudflare-env",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "vite build --app",
+		"build:custom-mode": "vite build --app -c vite.config.custom-mode.ts",
+		"check:types": "tsc --build",
+		"dev": "vite dev",
+		"dev:custom-mode": "vite dev -c vite.config.custom-mode.ts",
+		"preview": "vite preview",
+		"preview:custom-mode": "vite preview -c vite.config.custom-mode.ts"
+	},
+	"devDependencies": {
+		"@cloudflare/vite-plugin": "workspace:*",
+		"@cloudflare/workers-tsconfig": "workspace:*",
+		"@cloudflare/workers-types": "^4.20241230.0",
```

Comment:
Is the idea that dependabot will update this across the repo so it will stay in sync without being in the catalog?

Reply:
That's the idea. If it doesn't work for the playground, we can revisit.